sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/FormRecognizerTestEnvironment.cs

```diff
@@ -61,91 +45,47 @@ public FormRecognizerTestEnvironment() : base("formrecognizer")
         public string TargetResourceRegion => GetRecordedVariable(TargetResourceRegionEnvironmentVariableName);
 
         /// <summary>
-        /// The name of the directory where the running assembly is located.
+        /// The absolute path of the directory where the running assembly is located.
         /// </summary>
-        /// <value>The name of the current working directory.</value>
+        /// <value>The absolute path of the current working directory.</value>
         private static string CurrentWorkingDirectory => Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
 
         /// <summary>
-        /// The relative path to the JPG file which contains the form to be used for tests.
-        /// </summary>
-        /// <value>The relative path to the JPG file.</value>
-        public static string FormPath => CreatePath(FormFilename);
-
-        /// <summary>
-        /// The URI string to the JPG file which contains the form to be used for tests.
+        /// Creates an absolute path to a file contained in the local test assets folder.
         /// </summary>
-        /// <value>The URI string to the JPG file.</value>
-        public static string FormUri => CreateUri(FormFilename);
+        /// <param name="filename">The name of the file to create a path to.</param>
+        /// <returns>An absolute path to the specified file.</returns>
+        public static string CreatePath(string filename) =>
+            Path.Combine(CurrentWorkingDirectory, AssetsFolderName, filename);
```

Comment:
Trying to simplify things as much as I can, so I removed the `assetFolder` parameter since it's very unlikely that we'll be using it. Pretty easy to put it back in place if we change our minds.

Same reasoning applies to `CreateUri` and the `fileUriFormat` parameter.